test/04-translations.test.ts

```diff
@@ -47,9 +47,9 @@ describe('04 native translation tests', function () {
 					apiPrefix: 'v1/',
 					resource: 'student',
 					body: {
-						matrix_number: faker.datatype.number({ min: 100000 }),
-						name: faker.name.firstName(),
-						lastname: faker.name.lastName(),
+						matrix_number: faker.number.int({ min: 100000, max: 2 ** 31 }),
```

Comment:
Previously `max` defaulted to `min + 99999` and now it defaults to `Number.MAX_SAFE_INTEGER` which is bigger than a postgres int, so I changed the max to fit within a postgres int